includes/utils/class-amp-dom-utils.php

```diff
@@ -365,31 +365,22 @@ public static function has_class( DOMElement $element, $class ) {
 	 * If the element does not have an ID, create one first.
 	 *
 	 * @since 1.4.0
+	 * @since 1.5.1 Deprecated for AmpProject\Dom\Document::getElementId()
+	 *
+	 * @deprecated Use AmpProject\Dom\Document::getElementId() instead.
 	 *
 	 * @param DOMElement $element Element to get the ID for.
-	 * @param string     $prefix  Optional. Defaults to '_amp_wp_id_'.
+	 * @param string     $prefix  Optional. Defaults to 'amp-wp-id'.
```

Comment:
Should we use a common AMP prefix here? e.g.: `i-amp-`?

Reply:
//cc @westonruter

Reply:
I'd love to use something more official for this, but we'll have to make sure that all logic takes existing Ids into account, then. Otherwise, JS logic might use same Ids than PHP logic, for example.

Reply:
Default prefix was changed to `i-amp` via https://github.com/ampproject/amp-wp/pull/4799